egs2/m4singer/svs1/local/data_prep.py

```diff
@@ -0,0 +1,220 @@
+import argparse
+import json
+import os
+import shutil
+
+import librosa
+import miditoolkit
+import numpy as np
+
+from espnet2.fileio.score_scp import SingingScoreWriter
+
+"""Generate segments according to structured annotation."""
+"""Transfer music score into 'score' format."""
+
+SELECTED_SINGER = [
+    "Alto-1",
+    "Alto-2",
+    "Alto-3",
+    "Alto-4",
+    "Alto-5",
+    "Alto-6",
+    "Bass-1",
+    "Bass-2",
+    "Bass-3",
+    "Soprano-1",
+    "Soprano-2",
+    "Soprano-3",
+    "Soprano-4",
+    "Tenor-1",
+    "Tenor-2",
+    "Tenor-3",
+    "Tenor-4",
+    "Tenor-5",
+    "Tenor-6",
```

Comment:
Can you verify that the selected singer is the official test set?

Reply:
These are the singers in M4Singer. We can choose singers from this list according to different condition. It seems that there is no official split for test set.

Reply:
sounds good